app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
```

Comment:
These points don't apply strongly in this particular case because this is a few lines of code in a student app, but generally speaking, merging code that won't be needed in the foreseeable future seems like a bad practice for the following reasons:

- It adds to the complexity of the app and the burden associated with maintaining it. The excess code can break, causing exceptions, and it has associated tests, so even if the breakdown is more subtle, it could hold up a deployment or a merge.
- If the code is user-facing, which this is, it could easily result in bad/confusing UX.